tests/test_cli.py

```diff
@@ -197,9 +197,14 @@ def test_log_encoding(self, caplog, monkeypatch):
                 self.exec_cli(command)
 
         # first message should be 'log encoding: <encoding>'
-        match = caplog.records()[0].message.split(':')
+        match = None
+        for record in caplog.records():
```

Comment:
Why not check first message received here? Log encoding should be the first record, this test was designed to ensure that